e2e/package.json

```diff
@@ -64,5 +64,8 @@
   "license": "Apache-2.0",
   "engines": {
     "yarn": ">= 1.0.0"
+  },
+  "dependencies": {
+    "ndarray": "^1.0.19"
```

Comment:
We already have `tf.tensor` for high-dimensional arrays. Can we use that instead?